configure.ac

```diff
@@ -281,13 +275,6 @@ AS_IF([test x$use_smime != "xno"], [
 	SMIMEAUX_TARGET="smime_keys"
 ])
 
-dnl --enable-sidebar
-AS_IF([test x$use_sidebar = "xyes"], [
-	AC_DEFINE(USE_SIDEBAR, 1, [Define if you want support for the sidebar.])
```

Comment:
By removing this block, you also removed the corresponding #define USE_SIDEBAR 1 in config.h, that's why you get the errors you mentionned, because some symbols (like SidebarWidth) are defined only if USE_SIDEBAR is defined. I'll push a fixup commit, feel free to squash it in your own commit or even move that line if it makes more sense somewhere else.